src/pages/Privacy.tsx

```diff
@@ -0,0 +1,730 @@
+import { Container, Link, Typography } from '@mui/material';
+import { genMailToLink } from '../lib/mail';
+
+export default function Privacy(): JSX.Element {
+  const contactLink = (
+    <Link href={genMailToLink({ subject: 'Privacy Notice' })}>
+      info@dreamscholars.org
+    </Link>
+  );
+
+  return (
+    <Container maxWidth="md" sx={{ p: 2 }}>
+      <Typography variant="h3" gutterBottom>
+        Privacy Notice
+      </Typography>
+      <Typography variant="h6" paragraph>
+        Last Updated: April 25, 2022
+      </Typography>
+
+      <Typography paragraph>
+        This privacy notice for DreamScholars ("<strong>Company</strong>," "
+        <strong>we</strong>," "<strong>us</strong>," or "<strong>our</strong>"),
+        describes how and why we might collect, store, use, and/or share ("
+        <strong>process</strong>") your information when you use our services ("
+        <strong>Services</strong>"), such as when you:
```

Comment:
is the ("Services") needed? 
What about the other words above in parens like:
("company", "we", "us", or "our") & ("process")

Reply:
I think it's for technicality.